Doc/library/http.client.rst

```diff
@@ -115,6 +115,22 @@ The module provides the following classes:
       The *strict* parameter was removed. HTTP 0.9 style "Simple Responses" are
       no longer supported.
 
+This module provides the following function:
+
+.. function:: parse_headers(fp)
+
+   Parse the headers from a file pointer *fp* representing a HTTP request/response.
```

Comment:
max line length should be 80 characters.
https://devguide.python.org/documenting/#style-guide